generative_data_prep/data_prep/pipeline.py

```diff
@@ -402,6 +402,9 @@ def pipeline_main(  # noqa: C901
         test_ratio,
     )
 
+    lines = open(input_file_path).readlines()
+    assert len(lines) >= num_training_splits + num_dev_splits + num_test_splits, "Number of splits exceeds number of entries."
```

Comment:
This is a customer facing repo, please add helpful error that explains the problem more clearly and how to fix it